configs/default/IFRC-JBF7.config

```diff
@@ -0,0 +1,212 @@
+# version
```

Comment:
The `# version` line needs to be removed.

Reply:
Thank you for your reminder, it has been corrected.